app/Console/Commands/ClearOutFailedGitHubInvites.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\External\GitHub\GitHubApi;
+use App\External\HasApiProgressBar;
+use App\External\WooCommerce\Api\WooCommerceApi;
+use App\Models\Customer;
+use App\Notifications\GitHubInviteExpired;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Notification;
+use Illuminate\Support\Str;
+
+class ClearOutFailedGitHubInvites extends Command
+{
+    use HasApiProgressBar;
+
+    protected $signature = 'denhac:clear-out-failed-git-hub-invites {--dry-run}';
+
+    protected $description = 'Members can add their GitHub username to their profile, but they don\'t always accept the
+    invite and it expires after 7 days. GitHub keeps those and we can check against our customer\'s. If there\'s a
+    failed invite, we update their profile to remove the GitHub username (they can add it back) and email them if they
+    are a current member letting them know about the issue.';
+
+    private GitHubApi $gitHubApi;
+    private WooCommerceApi $wooCommerceApi;
+
+    public function __construct(
+        GitHubApi      $gitHubApi,
+        WooCommerceApi $wooCommerceApi
+    )
+    {
+        parent::__construct();
+
+        $this->gitHubApi = $gitHubApi;
+        $this->wooCommerceApi = $wooCommerceApi;
+    }
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
```

Comment:
This function was a bit difficult to parse even knowing what it was trying to accomplish.

`$inOrganization = $this->gitHubApi->denhac()->list()->map()` was the most difficult to read.

What does `denhac` do? What does `list` return? What is a `$ghm`? What's at `login`?
I can _guess_, but prefixing with 
`// list of github usernames for all members of the denhac github org`
and/or using more verbose variable names would have helped a lot.

Reply:
`denhac` is a shortcut to `organization("denhac")` and I was trying to decide whether it should be `denhac_organization` or what. `denhac_org` came to mind but that felt like a `denhac.org` thing or something. I just was hoping for it to be somewhat terse but descriptive enough. $ghm is "GitHub members" but honestly that was when it was grabbing from the member's team and not the organization itself. Let me refactor that code a little, especially because I just noticed a bug in it 😆

Reply:
I think this is the only one I'm still unsure on. Let me know if those code changes helped with readability.